src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -218,31 +225,37 @@ export default class Resources extends Module {
 	}
 
 	_onEvent(event) {
-		if (event.amount === 0) {
-			// Misses or attacks against invuln targets do not trigger resource gain
-			return
-		}
+		const _actionHit = (event.HitType !== HitType.MISS && event.HitType !== HitType.IMMUNE)
```

Comment:
```suggestion
		const _actionHit = (event.hitType !== HitType.MISS && event.hitType !== HitType.DODGE)
```

I _think_ `IMMUNE` is a wow or eso thing - unless you've seen ff logs with it?

Reply:
https://www.fflogs.com/v1/report/events/damage-done/z8HnpaXZMkWGdvj3?start=37735832&end=38171812&sourceid=362

https://www.fflogs.com/reports/z8HnpaXZMkWGdvj3#fight=49&type=damage-done&source=362&view=events&ability=7519

Based on results of Omega M/F and hitting the "wrong" target with Contre Sixte, it appears that Invulnerable is being mapped to the Immune hit type.

Reply:
Would it make more sense to extend the 'damage' and 'aoedamage' event types to have a SuccessfulHit (or UnSuccessfulHit) property, and put a const array of "hit failed" types in core?  I think that would save a lot of potential duplication by consumers, but I'm not sure on where I would need to add that logic to the base 'damage' event.

Reply:
>Based on results of Omega M/F and hitting the "wrong" target with Contre Sixte, it appears that Invulnerable is being mapped to the Immune hit type.

Huh, you're right. I must've totally missed that in my log scouring when I wrote that initial mapping, thanks for the catch.